https/tls-config.yml

```diff
@@ -0,0 +1,28 @@
+#TLS CONFIG YAML
```

Comment:
An example config should work

Reply:
Since there is no method for having a preset cert & key is it worth removing the example config?

Reply:
I think it's useful to have it, but let's make sure it's syntactically valid